src/common/meta/dashboards/v4/mod.rs

```diff
@@ -211,6 +213,19 @@ pub struct DrillDown {
     data: Option<DrillDownData>,
 }
 
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct MarkLine {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "type")]
+    typee: Option<String>,
```

Comment:
@ktx-abhay Typee or Type ?

Reply:
`type` is a reserved keyword in Rust. Therefore, I use `#[serde(rename = "type")]`, which renames `typee` to `type`.